spring-integration-core/src/test/java/org/springframework/integration/message/TestHandlers.java

```diff
@@ -89,4 +90,10 @@ public final static Object countingCountDownHandler(final AtomicInteger counter,
 		};
 	}
 
+	public static class RequestHeaderCopyingEchoHandler extends AbstractReplyProducingMessageHandler {
```

Comment:
I'd move this class to the `MessageBuilderAtConfigTests` as `private` and forget. What is good, it still works for `<bean>`
Plus update, please, Copyright for all affected classes and add yourself to the `@author`.
Thanks